main.js

```diff
@@ -0,0 +1,23 @@
+const navMenu = document.querySelector('.menu');
+const menuIcon = document.querySelector('.ambony');
+const menuLinks = document.querySelectorAll('.menu-item');
+
+menuIcon.addEventListener('click', () => {
+  const visibilityMenu = navMenu.getAttribute('item-visibility');
+  const visibilityIcon = menuIcon.getAttribute('cancle-visibility');
+
+  if (visibilityMenu === 'false' && visibilityIcon === 'false') {
+    navMenu.setAttribute('item-visibility', 'true');
+    menuIcon.setAttribute('cancle-visibility', 'true');
+  } else {
+    navMenu.setAttribute('item-visibility', 'false');
+    menuIcon.setAttribute('cancle-visibility', 'false');
+  }
+});
```

Comment:
Kindly include logic to close the menu when any of the menu items is clicked and also check that the appropriate section of the page is navigated to.